ecosystem/typescript/sdk/src/providers/indexer.ts

```diff
@@ -310,4 +318,98 @@ export class IndexerClient {
     };
     return this.queryIndexer(graphqlQuery);
   }
+
+  /**
+   * Queries account's current tokens.
+   * This query queries for any type of tokens (i.e NFT, Fungiable, soul bound, etc).
+   *
+   * @returns GetAccountTokensQuery response type
+   */
+  async getAccountTokens(ownerAddress: MaybeHexString, options?: PaginationArgs): Promise<GetAccountTokensQuery> {
+    const address = HexString.ensure(ownerAddress).hex();
+    IndexerClient.validateAddress(address);
+    const graphqlQuery = {
+      query: GetAccountTokens,
+      variables: { address, offset: options?.offset, limit: options?.limit },
+    };
+    return this.queryIndexer(graphqlQuery);
+  }
+
+  /**
+   * Queries all tokens of a specific collection that an account owns
+   *
+   * @param ownerAddress owner address
+   * @param collectionAddress the collection address
+   * @returns GetAccountCollectionTokensQuery response type
```

Comment:
Same as an earlier comment - Can we provide more details on the resp structure here or link to a doc that does?